automation/src/test/scala/org/broadinstitute/dsde/firecloud/fixture/DownloadFixtures.scala

```diff
@@ -0,0 +1,94 @@
+package org.broadinstitute.dsde.firecloud.fixture
+
+import com.typesafe.scalalogging.LazyLogging
+import java.io.File
+import java.nio.file.Files
+import java.nio.file.attribute.PosixFilePermission
+import java.time.LocalDateTime
+import java.time.format.DateTimeFormatter
+
+import org.broadinstitute.dsde.firecloud.component.Clickable
+import org.broadinstitute.dsde.workbench.service.util.Util
+import org.scalatest.concurrent.Eventually
+import java.util.UUID
+
+import org.broadinstitute.dsde.workbench.service.test.WebBrowserUtil
+import org.openqa.selenium.WebDriver
+
+trait DownloadFixtures extends Eventually with LazyLogging with WebBrowserUtil {
+
+
+  def makeTempDownloadDirectory(): String = {
+    /*
+     * This might work some day if docker permissions get straightened out... or it might not be
+     * needed. For now, we instead `chmod 777` the directory in run-tests.sh.
+    new File("chrome").mkdirs()
+    val downloadPath = Files.createTempDirectory(Paths.get("chrome"), "downloads")
+    val permissions = Set(PosixFilePermission.OWNER_WRITE, PosixFilePermission.GROUP_WRITE, PosixFilePermission.OTHERS_WRITE)
+    Files.setPosixFilePermissions(downloadPath, permissions.asJava)
+    downloadPath.toString
+     */
```

Comment:
does this comment block still need to be here?

Reply:
This was a huge pain to figure out and the way that it currently works is an ugly hack that is in fiab-start... far away from here where it matters... so I'd much rather be able to do something here. There might be a way to get it to work but I couldn't afford the time at the time.